### ISSUE TITLE
Touch/fav-icons

### ISSUE THREAD

**Issue by kdungs** [lhcb/starterkit#4]
Oh yeah. http://stackoverflow.com/questions/23849377/html-5-favicon-support

**Comment by kdungs** [comment]
@alexpearce can you do this from the `.svg` file?

**Comment by alexpearce** [comment]
Yes, but the SVG you sent me doesn't have the font shapes embedded, so it doesn't render correctly:

<img width="138" alt="Bad, bad logo" src="https://cloud.githubusercontent.com/assets/540853/15248693/e07a33a0-191b-11e6-87f3-23191ac07e34.png">

**Comment by kdungs** [comment]
Revisiting this after quite some time. Do you still need a better image file? :)

**Comment by alexpearce** [comment]
No, I have a `.ai` now I can work with.
